### PR TITLE
ci: trigger browser compatibility by comment

### PR DESCRIPTION
**Pull request by fi3ework**
implement the rest of #335, use `/tier1` comment to trigger the workflow.

### CODE REVIEW

**Comment by fi3ework** [comment]
/tier1 why does it not run? 😕

**Comment by lawvs** [comment]
I think `workflow_dispatch` is enough

Screenshot 2023-01-21 at 1 25 21 PM

**Comment by fi3ework** [comment]
@lawvs Yeah, that's the correct trick.

**Comment by fi3ework** [comment]
I would like to demonstrate the workflow:
1. comment on a PR that starts with `/tier1` will trigger a `rocket` emoji reaction to mark the workflow has run.
2. the comment-trigger workflow will use `createWorkflowDispatch` to dispatch a new workflow for `browser-compatibility.yml`. The trick here is to pass the PR's owner repo name and ref as the inputs to the workflow. We can't directly launch a job in PR author's repo, and neither launch an unexisting ref in `toeverything/blocksuite` (I tried some Git ref expression like `refs/heads/fi3ework:feat-branch`, but not work, did I missed something? 🤔).
3. `browser-compatibility.yml` will use the inputs to checkout repository of PR's repo and branch. Then run the test commands before PR is merged.

I tried in my demo repo: [PR](https://github.com/tinker7th/test-ci/pull/4), [trigger workflow](https://github.com/tinker7th/test-ci/actions/runs/3975862069), [test workflow](https://github.com/tinker7th/test-ci/actions/runs/3975863102/jobs/6815986128)

File: .github/workflows/browser-compatibility.yml [hunk 1]
```diff
@@ -4,7 +4,18 @@ on:
...
+      repo:
+        description: 'blocksuite repository to use'
+        required: true
+        type: string
+        default: 'toeverything/blocksuite'
+      ref:
+        description: 'blocksuite branch to use'
+        required: true
+        type: string
+        default: 'main'
```
Comment (lawvs) [inline .github/workflows/browser-compatibility.yml:18]:
```suggestion
        default: 'master'
```

**Review by doodlewind — APPROVED** [review]
LGTM

**Pull request merged**